Streetcode/Streetcode.WebApi/appsettings.json

```diff
@@ -32,5 +32,11 @@
   "Instagram": {
     "InstagramID": "NiceIdForInstagram",
     "InstagramToken": "SuperNiceTokenForInstagram"
+  },
+  "CORS": {
+    "AllowedOrigins": [ "*" ],
+    "AllowedHeaders": [ "*" ],
+    "AllowedMethods": [ "*" ],
```

Comment:
Let's allow nothing by default.
In Local config let's allow localhost.
In Stage and Prod we will inject this through Env variables.